docs/setup/aks/getting-started-aks.md

```diff
@@ -47,7 +48,9 @@ az aks create --resource-group my-nuclio-k8s-rg --name myNuclioCluster --node-co
 
 After several minutes, the deployment completes and returns information about the AKS deployment, in JSON format.
 
-**Install the kubectl CLI**, unless you already have it installed (in which case you can skip to the next step). The [`kubectl`](https://kubernetes.io/docs/user-guide/kubectl-overview/) Kubernetes command-line application enables you to connect to the Kubernetes cluster from your client computer. To install `kubectl` locally, run the following `az` command (see the [Azure CLI documentation](https://docs.microsoft.com/cli/azure/aks#az_aks_install_cli)):
+**Install the kubectl CLI**, if it's not already installed (in which case you can skip to the next step).
```

Comment:
```suggestion
**Install the kubectl CLI**, if it's not already installed, in which case you can skip to the next step.
```

Reply:
I prefer my version, or this alternative:
```markdown
**Install the kubectl CLI**, If the CLI is already installed, you can skip to the next step.
```

Q: In looking at this doc, I think it's unclear what's the "next step" to skip to here (if the CLI is already installed). Because we don't have marked or enumerated "steps". I suspect that the intention is just to skip the installation and move on to connecting the cluster with kubectl, but one might think that they can skip to the next Markdown section. => I would enumerate the steps in this section ("Set up your AKS cluster"). @omesser please confirm before I make the changes.

Reply:
@Sharon-iguazio enumeration sounds good to me

Reply:
1. I rephrased per my suggestion in the previous comment (+ I split it into 2 source lines) — commit 378d3d5:
    ```markdown
    **Install the kubectl CLI**.
    If the CLI is already installed, you can skip to the next step.
    ```
2. I enumerated the steps, and make the "next step" reference in Step 3 (kubectl CLI installation) a link to Step 4 — commit 2a40378.
=> @omesser / @liranbg please approve.